internal/proto/opamp.proto

```diff
@@ -538,6 +538,10 @@ enum AgentCapabilities {
     AcceptsOtherConnectionSettings = 0x00000200;
     // The Agent can accept restart requests.
     AcceptsRestartCommand          = 0x00000400;
+    // The Agent reports Health
+    ReportsHealth                  = 0x00000800;
+    //    // The Agent will report RemoteConfig in AgentToServer.
+    //    ReportsRemoteConfig            = 0x00001000;
```

Comment:
We should consider adding this to be consistent in Accepts/Reports pairing of capabilities.